osbs/build/build_request.py

```diff
@@ -770,6 +770,11 @@ def render(self, validate=True):
             self.template['spec']['output']['to']['name'] = \
                 self.spec.image_tag.value
 
+        if self.spec.build_imagestream.value:
+            self.template['spec']['strategy']['customStrategy']['from']['kind'] = 'ImageStreamTag'
+            self.template['spec']['strategy']['customStrategy']['from']['name'] = \
+                self.spec.build_imagestream.value
```

Comment:
I'd move this near the code that sets `build_image` so that it's clear in the code that the options are mutually exclusive.